### PR TITLE
Update npm modules

### PR DESCRIPTION
**Pull request by Brouilles**
Breaking changes in pidusage 2.x.x. So, not updated.

### CODE REVIEW

File: package.js [hunk 1]
```diff
@@ -6,11 +6,11 @@ Package.describe({
 });
 
 var npmModules = {
-  "debug": "0.7.4",
+  "debug": "4.1.1",
```
Comment (zodern) [inline package.js]: New versions of debug use const/let, which breaks using in Meteor 1.4 and 1.5.
https://travis-ci.org/monti-apm/monti-apm-agent/jobs/557454465
https://travis-ci.org/monti-apm/monti-apm-agent/jobs/557454466

Comment (Brouilles) [reply package.js]: I see. I forgot Meteor 1.4 and 1.5 doesn't support ES6. I think it's the same for Lru-cache.

Comment (Brouilles) [reply package.js]: So, I switch to debug 0.8.1 (Apr 15, 2014) and lru-cache 4.1.5.

**Comment by Brouilles** [comment]
Hi, it's not good for you?
You want new changes?
Thanks!

**Comment by zodern** [comment]
Thanks @Brouilles

**Pull request merged**